tests/unit/modules/test_virt.py

```diff
@@ -1871,7 +1871,7 @@ def test_update(self):
         )
         setxml = ET.fromstring(define_mock.call_args[0][0])
         self.assertEqual("2", setxml.find("vcpu").text)
-        self.assertEqual("2048", setxml.find("memory").text)
+        self.assertEqual("2147483648", setxml.find("memory").text)
```

Comment:
Can you please submit this fix in a separate PR.
I'd like to merge that sooner than this PR.
Thanks!